operator/cmd/mesh/operator-common.go

```diff
@@ -82,3 +94,113 @@ tag: {{.Tag}}
 	manifest, err := r.RenderManifest(vals)
 	return vals, manifest, err
 }
+
+var k8sRESTConfig *rest.Config
+
+var k8sClientset *kubernetes.Clientset
+
+var currentKubeconfig string
+
+var currentContext string
```

Comment:
can we remove these? can be in another pr

Reply:
done.